src/screens/BrowseSourceScreen/BrowseSourceScreen.tsx

```diff
@@ -131,6 +131,10 @@ const BrowseSourceScreen = ({ route, navigation }: BrowseSourceScreenProps) => {
                 libraryStatus={inLibrary}
                 onPress={() => navigateToNovel(item)}
                 isSelected={false}
+                addSkeletonLoading={
+                  (hasNextPage && !searchText) ||
+                  (hasNextSearchPage && !!searchText)
```

Comment:
```suggestion
                  (hasNextSearchPage && searchText)
```

Reply:
It would work as how you suggest, but also throw a ts error (can't assign string to boolean). An other option would be to use Boolean().

Reply:
oh ye, I forgot it. But I think using Boolean is better. It is a bit longer but more readable I think